angular-app/src/main.ts

```diff
@@ -4,6 +4,8 @@ import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 import { AppModule } from './app/app.module';
 import { environment } from './environments/environment';
 
+import { FormsModule } from '@angular/forms';
```

Comment:
What is this import being used for?

Reply:
Its used on the main feed for the text input for making a post

Reply:
Does it need to be imported here? It looks like it was already also imported in app.module.ts.